platform/broadcom/sswsyncd/Makefile

```diff
@@ -0,0 +1,36 @@
+CC = g++
+CFLAGS_COMMON = -ansi -fPIC -std=c++11 -Wall -Wcast-align -Wcast-qual \
+    -Wconversion -Wdisabled-optimization -Werror -Wextra -Wfloat-equal \
+    -Wformat=2 -Wformat-nonliteral -Wformat-security -Wformat-y2k \
+    -Wimport -Winit-self -Winline -Winvalid-pch -Wlong-long \
+    -Wmissing-field-initializers -Wmissing-format-attribute \
+    -Wmissing-include-dirs -Wmissing-noreturn -Wno-aggregate-return \
+    -Wno-padded -Wno-switch-enum -Wno-unused-parameter -Wpacked \
+    -Wpointer-arith -Wredundant-decls -Wshadow -Wstack-protector \
+    -Wstrict-aliasing=3 -Wswitch -Wswitch-default -Wunreachable-code \
+    -Wunused -Wvariadic-macros -Wwrite-strings -Wno-switch-default \
+    -Wconversion -Wlong-long
+
+CFLAGS = -g -std=c++11
```

Comment:
Can you change this to `+=`, and modify the linker commands below to include `$(CPPFLAGS) $(LDFLAGS)`?

This is so that the flags that `dpkg-buildpackage` sets via `dpkg-buildflags` can come through and take effect. For reference, these are the relevant flags:

```
CFLAGS=-g -O2 -ffile-prefix-map=/=. -fstack-protector-strong -Wformat -Werror=format-security
CPPFLAGS=-Wdate-time -D_FORTIFY_SOURCE=2
CXXFLAGS=-g -O2 -ffile-prefix-map=/=. -fstack-protector-strong -Wformat -Werror=format-security
...
LDFLAGS=-Wl,-z,relro
```